algorithm-registry.md

```diff
@@ -22,7 +23,9 @@ for at least one of these algorithms.
 |           | rsa-sign-pss-4096-sha256   | sign/verify |                                                                                  |
 | ECDSA     | ecdsa-sha2-256-nistp256    | sign/verify |                                                                                  |
 |           | ecdsa-sha2-384-nistp384    | sign/verify |                                                                                  |
+|           | ecdsa-sha2-256-nistp384    | sign/verify | Not recommended.                                                                 |
 |           | ecdsa-sha2-512-nistp521    | sign/verify |                                                                                  |
+|           | ecdsa-sha2-256-nistp521    | sign/verify | Not recommended.                                                                 |
```

Comment:
NB: I added these because they already exist in the ecosystem (i.e. because of clients using SHA256 regardless of the NIST curve chosen). However, if the PGI no longer accepts them, perhaps I should mark them as "verify only" and add a clarifying note? Curious what people think.

Reply:
I'm fine with sign/verify for now, but can we clarify why they aren't recommended?

Reply:
I think that verify only would make sense, seems like a small change. +1 on clarify why they are not recommended.